Riot/ViewController/RoomSettingsViewController.m

```diff
@@ -2081,6 +2081,8 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
             topicTextView.editable = (oneSelfPowerLevel >= [powerLevels minimumPowerLevelForSendingEventAsStateEvent:kMXEventTypeStringRoomTopic]);
             topicTextView.textColor = kRiotSecondaryTextColor;
             
+            topicTextView.keyboardAppearance = kRiotKeyboard;
```

Comment:
If I understood correctly, we don't need to set keyboardAppearance here, because it is handled by [UITextField appearance], isn't it?

Reply:
I would think it would but in my testing, it doesn't change to be the dark keyboard unless this line of code is there. I'm not exactly sure why.

Reply:
I realized this is not a UITextField but a UITextView.
Could you please test again by setting `[UITextView appearance].keyboardAppearance` in `[RiotDesignValues userInterfaceThemeDidChange]` method.

Reply:
I don't think that is possible. That crashes the app. I don't really like doing the kRiotKeyboard thing but as far as I can tell that is the only way to get it to work on HPGrowingTextView and the UITextView.

https://stackoverflow.com/questions/21503906/uitextview-keyboardappearance-property-to-set-uikeyboardappearancedark